### PR TITLE
Implement searchLabel prop for custom translation

### PR DESCRIPTION
**Pull request by PatricNox**
Now the end-user can pass an optional prop to override the input placeholder.

image

### CODE REVIEW

**Comment by PatricNox** [comment]
May need some testing before merging, this is something my customer desires so kind of need a release of this asap

**Comment by PatricNox** [comment]
force pushes is due to wrong commit author (multiple git configs), sorry for that. 

> Tip: merge with squash

File: src/components/Emoji.vue [hunk 1]
```diff
@@ -126,6 +126,10 @@ export default {
     height: {
       type: String,
       default: "250px"
+    },
+    searchLabel: {
+      type: String,
+      default: "Search"
```
Comment (PatricNox) [inline src/components/Emoji.vue:132]: @vestusola  i think i missed something here, this seems to have broken the app

Comment (vestusola) [reply src/components/Emoji.vue:132]: Oops!

**Pull request merged**